arangodb-net-standard.Test/CursorApi/CursorApiClientTest.cs

```diff
@@ -342,7 +346,7 @@ public async Task DeleteCursorAsync_ShouldThrow_WhenCursorDoesNotExist()
         [Fact]
         public async Task DeleteCursorAsync_ShouldSucceed()
         {
-            var response = await _cursorApi.PostCursorAsync<long>("FOR i IN 0..1000 RETURN i");
+            var response = await _cursorApi.PostCursorAsync<long>("FOR i IN 0..1000 RETURN i", null);
```

Comment:
Minor: the null bindVars can now be removed.

Reply:
Done. Thanks :-)